examples/magento-graphcms/pages/[...url].tsx

```diff
@@ -75,7 +75,9 @@ function CategoryPage(props: CategoryProps) {
         metaDescription={page?.metaDescription}
         metaRobots={page?.metaRobots.toLowerCase().split('_') as MetaRobots[]}
         canonical={page?.url ? `/${page.url}` : undefined}
-        {...category}
+        name={category?.name}
+        meta_title={category?.meta_title}
+        meta_description={category?.meta_description}
```

Comment:
The CategoryMeta component should `Omit<PageMetaProps, 'title' | 'metaDescription' | 'children'>`. Then we can keep the `{...category}`

(It is a bit unfortunate that Magento opted to use `children` as the field which makes handling these situations a bit weird.)

Reply:
Yep, that's better, omitting children was enough 👌